src/main/resources/db/changelog/db.changelog-RDM-6916.xml

```diff
@@ -0,0 +1,43 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+
+    <!-- conditional drop to remove index manually created in some environments -->
+    <changeSet id="rdm-6916-drop" author="matt.nayler@hmcts.net">
+        <preConditions onFail="MARK_RAN">
+            <indexExists indexName="idx_case_event_created_date"
+                   schemaName="public"
+                   tableName="case_event" />
+        </preConditions>
+        <dropIndex indexName="idx_case_event_created_date"
+                   schemaName="public"
+                   tableName="case_event" />
+    </changeSet>
+
+
+    <changeSet id="rdm-6916" author="matt.nayler@hmcts.net">
+
+        <createIndex indexName="idx_case_event_created_date"
```

Comment:
Suggest you use Postgres' CONCURRENTLY modifier when creating these indexes; standard index creation will lock the case_event table for writes until the indexes are built, which could take time since it's a big table. You'd probably need to write <sql>, don't think liquibase supports the concurrently flag.

Reply:
Many thanks for your comment.  Some [timing details from a series of test runs](https://tools.hmcts.net/jira/browse/RDM-6916?focusedCommentId=739903&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-739903) have been added to the Jira ticket associated to this PR and will be used during the review and checkpoint process to decide the way forward.

NB: A separate set of scripts implementing the CONCURRENTLY modifier as you have recommended have been prepared in advance if the change is required.